_data/notices.yml

```
@@ -22,12 +22,12 @@
  #   messages:
  #      - message: |
  #          Due to ISP maintenance outage is expected during the mentioned time
-         
-  # - title: Possible service interruption due to Galaxy upgrade to v24.2 scheduled on 15-03-2025, between 12:00:00 (CET) and 14:00:00 (CET)
-  #   class: warning  # one of [success, info, warning, danger]
-  #   messages:
-  #      - message: |
-  #          Due to updates to the configuration, a short outage is possible during the mentioned time. 
+        
+ - title: Possible service interruption due to Galaxy upgrade to v25.0 scheduled on Monday 13 October 2025, between 13:00:00 (CET) and 17:00:00 (CET).
```

```suggestion
 - title: Possible service disruption due to Galaxy upgrade to v25.0 scheduled on Monday 13 October 2025, between 13:00:00 (CET) and 17:00:00 (CET).
```
Can we also expand the upgrade time window? Since the database migration can take a while.